diff -uraN webkitgtk-2.36.0/Source/WebKit/PlatformGTK.cmake omv-webkitgtk-2.36.0/Source/WebKit/PlatformGTK.cmake
--- webkitgtk-2.36.0/Source/WebKit/PlatformGTK.cmake	2022-03-16 14:48:02.000000000 +0100
+++ omv-webkitgtk-2.36.0/Source/WebKit/PlatformGTK.cmake	2022-03-30 23:20:38.531248900 +0200
@@ -675,7 +675,7 @@
     add_custom_target(WebKit2-${WEBKITGTK_API_VERSION}-gir
         DEPENDS WebKit
         DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
-        COMMAND CC=${CMAKE_C_COMPILER} CFLAGS="-Wno-deprecated-declarations ${CMAKE_C_FLAGS}" LDFLAGS=
+        COMMAND CC=${CMAKE_C_COMPILER} CFLAGS="-Wno-deprecated-declarations -fno-lto" LDFLAGS="-fno-lto"
             ${LOADER_LIBRARY_PATH_VAR}="${INTROSPECTION_ADDITIONAL_LIBRARY_PATH}"
             ${INTROSPECTION_SCANNER}
             --quiet
@@ -721,7 +721,7 @@
         OUTPUT ${CMAKE_BINARY_DIR}/WebKit2WebExtension-${WEBKITGTK_API_VERSION}.gir
         DEPENDS ${CMAKE_BINARY_DIR}/JavaScriptCore-${WEBKITGTK_API_VERSION}.gir
         DEPENDS WebKit2-${WEBKITGTK_API_VERSION}-gir
-        COMMAND CC=${CMAKE_C_COMPILER} CFLAGS="-Wno-deprecated-declarations ${CMAKE_C_FLAGS}"
+        COMMAND CC=${CMAKE_C_COMPILER} CFLAGS="-Wno-deprecated-declarations -fno-lto"
             LDFLAGS="${INTROSPECTION_ADDITIONAL_LDFLAGS}"
             ${LOADER_LIBRARY_PATH_VAR}="${INTROSPECTION_ADDITIONAL_LIBRARY_PATH}"
             ${INTROSPECTION_SCANNER}
